pkg/config/setup/config.go

```diff
@@ -1056,7 +1056,6 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("admission_controller.inject_auto_detected_libraries", true)                                                         // allows injecting libraries for languages detected by automatic language detection feature
 	config.BindEnv("admission_controller.auto_instrumentation.init_resources.cpu")
 	config.BindEnv("admission_controller.auto_instrumentation.init_resources.memory")
-	config.BindEnvAndSetDefault("admission_controller.auto_instrumentation.inject_all.namespaces", []string{})
```

Comment:
❓ 

If I understand correctly, this config option can be used to decide in which namespaces SSI should be injecting libraries. If this is no longer used, does this mean that enabling SSI will result in injecting libraries in all namespaces (asking because I don't see any restriction on namespaces during injection after the changes made in this PR)? Is this the intended behaviour?